README.md

```diff
@@ -1,4 +1,7 @@
 # docker
+
+![Release build](https://github.com/sgermanserrano/docker/workflows/Release%20build/badge.svg) ![Nightly build](https://github.com/sgermanserrano/docker/workflows/Nightly%20build/badge.svg) ![PR build](https://github.com/sgermanserrano/docker/workflows/PR%20build/badge.svg)
```

Comment:
Probably want to update these with the `autoware-ai` versions.

Reply:
Updated